SwiftWisdom/Core/Foundation/Not.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Not.swift
+//  bmap
+//
+//  Created by Logan Wright on 12/11/15.
+//  Copyright © 2015 Intrepid. All rights reserved.
+//
+
+/**
+ Inverts a given boolean return closure
+ 
+ Good for things like `array.filter(not(matchesThing))`
+ */
+public func not<T>(original: T -> Bool) -> T -> Bool {
```

Comment:
quick question, if we happen to include a library that exposes not as a global function, is there a way we can 'back out'?